openshift/deployment-template.yaml

```diff
@@ -119,3 +118,13 @@ parameters:
     name: IMAGE_STREAM_TAG
     value: 'latest'
 
+  - description: Namespace in which operator is serving events.
+    displayName: Operator namespace
+    required: true
```

Comment:
as these are required parameters, we need to change some ansible playbooks too... or.. maybe... by now we dont have a playbook/role deploying the operator? 

TODO: create workload operator provisioning playbook

Reply:
#6

Reply:
I think we are not deploying it yet. I deployed it manually for now into test environment with this template.